specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-06-01-preview/SourceControls.json

```diff
@@ -404,8 +417,41 @@
       ],
       "type": "object"
     },
+    "RepositoryAccess": {
```

Comment:
Schema should have a description or title.

Reply:
Fixed.